Resources/Maps/Shuttles/helix.yml

```diff
@@ -2775,13 +2775,13 @@ entities:
     - pos: -0.5,7.5
       parent: 1
       type: Transform
-- proto: SpawnPointMedicalCyborg
-  entities:
-  - uid: 401
-    components:
-    - pos: 7.5,6.5
-      parent: 1
-      type: Transform
+#- proto: SpawnPointMedicalCyborg
+#  entities:
+#  - uid: 401
+#    components:
+#    - pos: 7.5,6.5
+#      parent: 1
+#      type: Transform
```

Comment:
Does the map loader allow commented entities? Regardless the game code will bulldoze that uid etc.  Better to just yeet and delete the map entry tbh